### DISCUSSION TITLE
Trouble calling createMany on relation

### DISCUSSION THREAD

**Discussion by Kadajett** [prisma/prisma#15042] [Q&A]
Hi folks! I am relatively new to prisma. I am loving it so far, but I have run into an issue. I am trying to populate my users with data as its pulled from 3rd parties. I am getting lists of tokens, and I want to apply those tokens to the profile in the `ownedTokens` relation. 

I have a user model like this 
```prisma
model User {
    id              Int             @id @default(autoincrement())
    email           String?         @unique
    firstName       String?
    lastName        String?
    SocialAccounts  SocialAccount[]
    Playlists       Playlist[]
    WalletAddresses String          @unique

    ownedTokens               Token[]                     @relation(name: "userOwns")
    likedTokens               LikedToken[]
    LikedPlaylist             LikedPlaylist[]
    PlaylistAnnouncements     PlaylistAnnouncements[]
    PlaylistAnnouncementLikes PlaylistAnnouncementLikes[]
    UserBlogPost              UserBlogPost[]
    UserBlogPostLikes         UserBlogPostLikes[]
    UserBlogPostComments      UserBlogPostComments[]
    WalletAddress             WalletAddress[]
}
```

And my token model looks like this:
```prisma
model Token {
    id String @id

    createdAt         DateTime         @default(now())
    updatedAt         DateTime         @updatedAt
    userId            Int?
    ownerId           Int?
    likes             LikedToken[]
    tokenUrl          cloudflareAsset? @relation(fields: [cloudflareAssetId], references: [id])
    cloudflareAssetId Int?
    owner             User?            @relation(name: "userOwns", fields: [ownerId], references: [id])

    contractAddress String
    chain           String

    @@unique([contractAddress, chain, id])
}
```

My attempt to create the tokens looks like this: 
```ts
console.log("adding tokens to user");
await ctx.prisma.user.update({
  where: {
	  WalletAddresses: address,
  },
  data: {
	  ownedTokens: {
		  createMany: {
			  data: nfts.map((nft: token) => ({
				  contractAddress: nft.contract.address,
				  chain: Network.MATIC_MAINNET,
				  id: nft.tokenId,
				  token: nft.title,
  
				  cloudflareAssetId: 1,
			  })),
			  skipDuplicates: true,
		  },
	  },
  },
});
console.log("added tokens to user");
```

The type checking shows that its fine, but with the await there, it just stalls for some reason. Trying to figure out why. I am not getting any of the records in my DB, and I am not getting any error in my console. It just locks up. Any thoughts?

**Comment by austincrim** [comment]
Hey @Kadajett, glad you're enjoying Prisma!

What database provider are you using?

What version of Prisma are you using?

**Reply by vladi-stevanovic** [reply]
👋 Hello @Kadajett I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how! 

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
